tests/integration/test_charm.py

```diff
@@ -37,13 +37,14 @@ async def test_build_and_deploy(ops_test: OpsTest, kafka_charm):
         ops_test.model.deploy(kafka_charm, application_name=APP_NAME, num_units=1, series="jammy"),
     )
     await ops_test.model.wait_for_idle(apps=[APP_NAME, ZK_NAME])
-    assert ops_test.model.applications[APP_NAME].status == "waiting"
```

Comment:
Note: after this change we will need to update any integration tests involving Kafka (`kafka-bundle`, `data-integrator`, `kafka-test-app`)